src/Components/SocialLinks.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import './SocialLinks.css'
+import FacebookIcon from '../Assets/Images/facebook-icon.svg'
+import LinkedInIcon from '../Assets/Images/linkedin-icon.svg'
+import TwitterIcon from '../Assets/Images/twitter-icon.svg'
+import GitHubIcon from '../Assets/Images/github-icon.svg'
+
+function SocialLinks() {
+  return (
+    <div className='social-links-container'>
+      <ul>
+        <li>
+          <a href="#"><img src={FacebookIcon} alt="" /></a>
```

Comment:
Unless these pictures actually link to something, probably best to remove the anchor around them.
```suggestion
          <img src={FacebookIcon} alt="" />
```

Reply:
These will link somewhere as well, just need to get that all set up. When I have links for them I will place them in the code.